folding/utils/ops.py

```diff
@@ -133,11 +134,12 @@ def calc_potential_from_edr(
     Returns:
         float: potential energy
     """
+    runandlog=RunAndLog()
```

Comment:
are you running black?

Reply:
just ran it

Reply:
https://github.com/macrocosm-os/folding/pull/147/commits/ade8061845c174f6038a3c5359587fe849f46bca